schema_watcher.go

```diff
@@ -489,6 +489,16 @@ func (s *SchemaWatcher) FindEnumByName(enum protoreflect.FullName) (protoreflect
 	return res.FindEnumByName(enum)
 }
 
+// ResolvedSchema returns the resolved schema in the form of a
+// FileDescriptorSet. Until AwaitReady returns a non-nil status, the return
+// value of this function can be nil.
```

Comment:
I would add a blurb that the caller must _not_ mutate the returned file descriptor set. If the caller wants to perform mutations on the schema, it must first make a copy via `proto.Clone`.

Reply:
Done.